src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -0,0 +1,289 @@
+const simpleGit = require("simple-git");
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+const {
+  createAmplifyBranches,
+  startReleaseJob,
+  createAmplifyApp,
+  readBuildSpec,
+} = require("./amplifyUtils");
+const BRANCH_NAME = "chore-amplify-migration-change-permalinks";
+const ORGANIZATION_NAME = "isomerpages";
+
+interface AmplifyAppInfo {
+  appId: string;
+  repoName: string;
+  name: string;
+  repoPath: string;
+}
+
+const listOfRepos: [string, string][] = [
+  ["repo-name", "Human Friendly Name"], // Modify this line when running the script
+];
+
+async function main() {
+  for (const [repoName, name] of listOfRepos) {
+    try {
+      await migrateRepo(repoName, name);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+}
+
+async function migrateRepo(repoName: string, name: string) {
+  const homeDir = process.env.HOME;
+  const repoPath = `${homeDir}/isomer-migrations/${repoName}`;
```

Comment:
Hmm can we specify `~` instead of using an env var?

Reply:
Sadly, `~` is not supported in [fs](https://github.com/nodejs/node/issues/684)
For clarity, ill use os.homedir() instead [here](https://github.com/isomerpages/isomer-conversion-scripts/commit/0a96a49dc5e98446f40cde35eef7c626678971a1)